src/plugins/mails.js

```diff
@@ -1,13 +1,25 @@
-export const sendEcartInstructions = (situationId) => {
+const generateResultLines = (droits, customAmount) => {
+  return droits
+    .map(({ amount, label }) => {
+      return `
+      - ${customAmount || amount} pour la prestation « ${label} ».
+    `
+    })
+    .join("")
```

Comment:
Pourquoi mettre les sauts de lignes dans la chaine de caractères plutôt que dans le `join` ?
```suggestion
    .map(({ amount, label }) => `      - ${customAmount || amount} pour la prestation « ${label} »`)
    .join("")
```